shared/css/color.scss

```diff
@@ -203,6 +203,11 @@ $light_gray_800: #54595E;
 $light_gray_900: #3F444B;
 
 $light_affirmative_100: #E2F1E2;
+$light_affirmative_500: #3EA33E;
+
+$light_negative_500: #E5311A;
+
+$light_caution_500: #F9CB28;
```

Comment:
Standard named colors from the figma design doc that are used in the temperature widget.